plugins/parsers/registry.go

```diff
@@ -188,24 +188,20 @@ type Config struct {
 }
 
 // NewParser returns a Parser interface based on the given config.
+// DEPRECATED: Please instantiate the parser directly instead of using this function.
```

Comment:
Should we remove this function instead of deprecating it? It's not an external api so what is deprecating meant to do?

Reply:
Well it might be used by external plugins and as this function was exported I'm a bit hesitant to remove it. But same as for the two comments above... :-)